dracut-functions.sh

```diff
@@ -1023,3 +1023,24 @@ get_dev_module() {
     fi
     echo "$dev_drivers"
 }
+
+# Check if file is in PE format
+pe_file_format() {
+    if [[ $# -eq 1 ]]; then
+        dos_header=$(hexdump -s 0 -n 64 -e'64/1 "%_c"' "$1")
+        signature=$(hexdump -s 128 -n 4 -e'4 "%_c"' "$1")
```

Comment:
This is just wrong. The PE header can reside virtually anywhere. You have to examine the offset in the DOS header to find the PE header.
You should build the latest master of systemd to test this as its PE header starts at offset 64 and your code spits out the wrong alignment.

Why not use `objdump -p`? Or better yet: why not just use ukify and get rid of senseless duplication of effort and gain support for more arches in the process?

Reply:
Thank you for your feedback, indeed `objdump -p` is cleaner !
Changes applied